app/assets/src/components/views/amr_heatmap/AMRHeatmapVis.jsx

```diff
@@ -139,6 +152,7 @@ export default class AMRHeatmapVis extends React.Component {
 }
 
 AMRHeatmapVis.propTypes = {
+  maxValueUpdater: PropTypes.func,
   samplesWithAMRCounts: PropTypes.arrayOf(
```

Comment:
Also, I didn't catch this last PR, but I don't think samplesWithAMRCounts needs to be on the state in this component. You can just directly use the props.

A rule of thumb in React is to only add to `this.state` when you need to, and to use `this.props` directly when you can. It makes it easier to track where a given value "lives"...whether the source of truth is that component or if the value is passed in from the parent.